pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -146,6 +146,25 @@ func (s *USMSuite) TestEnableHTTPMonitoring() {
 	_ = setupTracer(t, cfg)
 }
 
+func (s *USMSuite) TestDisableUSM() {
+	t := s.T()
+
+	cfg := testConfig()
+	cfg.ServiceMonitoringEnabled = false
+	// Enabling all features, to ensure nothing is forcing USM enablement.
+	cfg.EnableHTTPMonitoring = true
+	cfg.EnableHTTP2Monitoring = true
+	cfg.EnableKafkaMonitoring = true
+	cfg.EnablePostgresMonitoring = true
+	cfg.EnableGoTLSSupport = true
+	cfg.EnableNodeJSMonitoring = true
+	cfg.EnableIstioMonitoring = true
+	cfg.EnableNativeTLSMonitoring = true
```

Comment:
consider having some "sugar syntax" function "EnableAllCaps" for USM to have a centralized place for all config knobs (it is just too easy to miss one, when you configure them explicitly in different places in code

Reply:
I need to think where is the best place to locate this utility, as every location can be just "forgotten"